config/clusters/awi-ciroh/prod.values.yaml

```diff
@@ -50,37 +50,27 @@ basehub:
                   kubespawner_override:
                     image: "quay.io/2i2c/awi-ciroh-image:79bb68961852"
                 devcon1:
-                  display_name: CNN for downscaling climate forcing variables
-                  default: true
-                  slug: "devcon1"
+                  display_name: DevCon24 workshop : CNN for downscaling climate forcing variables
```

Comment:
You can use a `description` field to display extra information here as well.

Reply:
CI is also failing because `:` is not allowed here in the content of the string. Either move this to the description, or put `"` around the whole string

Reply:
Fixed